README.md

```diff
@@ -1,9 +1,10 @@
 # The Libyui Testing Image
 
-[![Build Status](https://travis-ci.org/libyui/ci-libyui-container.svg?branch=master)](https://travis-ci.org/libyui/ci-libyui-container)
+[![Workflow Status](https://github.com/libyui/libyui/workflows/CI/badge.svg?branch=master)](
+https://github.com/libyui/libyui/actions?query=branch%3Amaster)
```

Comment:
Is this url correct? I expected something like *https://github.com/libyui/ci-libyui-container/actions?query=branch%3Amaster)*